examples/rapids/02-rapids-gpu-cluster.ipynb

```diff
@@ -0,0 +1,542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-10-26T17:00:38.727567Z",
+     "iopub.status.busy": "2021-10-26T17:00:38.727285Z",
+     "iopub.status.idle": "2021-10-26T17:00:38.730682Z",
+     "shell.execute_reply": "2021-10-26T17:00:38.730051Z",
+     "shell.execute_reply.started": "2021-10-26T17:00:38.727541Z"
+    }
+   },
+   "source": [
+    "# Use RAPIDS on a GPU Cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/dask.png\" alt=\"Dask Logo\" style=\"width: 300px;\"/><img src=\"https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/rapids.png\" alt=\"RAPIDS Logo\" style=\"width: 300px;\"/>\n",
+    "\n",
+    "[RAPIDS](https://rapids.ai/) is a collection of libraries that enable you to take advantage of NVIDIA GPUs to accelerate machine learning workflows. We can use RAPIDS together with Dask to distribute calculations over a cluster. Minimal changes are required to transition from familiar `pandas` and `scikit-learn` code!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "This example describes how to run a machine learning training workflow using the famous NYC Taxi Dataset. This dataset contains information about taxi trips in New York City. For the purposes of this example, we will be looking at the Yellow Taxi data from January 2019.\n",
+    "We will use this data to answer the following question:\n",
```

Comment:
And given the comment above I would take this out (and take out any text that is copied from the first)